python/lsst/ts/MTAlignment/mock/mock_t2sa.py

```diff
@@ -0,0 +1,926 @@
+                    self.load_sa_template_file,
+                    r"(?P<file_path>.*)",
+                ),
+                ("!LST", self.set_power, r"(?P<value>.*)"),
+                ("!MEAS_DRIFT", self.execute_drift, r"(?P<point_group>.*)"),
+                (
+                    "!MEAS_SINGLE_POINT",
+                    self.measure_single_point,
+                    r"(?P<collection>.*);(?P<point_group>.*);(?P<point_n>.*)",
+                ),
+                (
+                    "!PUBLISH_ALT_AZ_ROT",
+                    self.set_alt_az_rot,
+                    r"(?P<alt>.*);(?P<az>.*);(?P<rot>.*)",
+                ),
+                ("!SAVE_SA_JOBFILE", self.save_sa_jobfile, r"(?P<filename>.*)"),
+                (
+                    "!SET_REFERENCE_GROUP",
+                    self.set_reference_group,
+                    r"(?P<reference_group>.*)",
+                ),
+                (
+                    "!SET_WORKING_FRAME",
+                    self.set_working_frame,
+                    r"(?P<working_frame>.*)",
+                ),
+                ("?LSTA", self.get_laser_status, ""),
+                (
+                    "?OFFSET",
+                    self.write_point_group_offset,
+                    r"(?P<reference_group>.*);(?P<point_group>.*)",
+                ),
+                (
+                    "?POINT_DELTA",
+                    self.measure_point_delta,
+                    r"(?P<p1collection>.*);(?P<p1group>.*);(?P<p1>.*);"
+                    r"(?P<p2collection>.*);(?P<p2group>.*);(?P<p2>.*)",
+                ),
+                ("?POS CAM", self.write_point_group_position_cam, ""),
+                ("?POS M1M3", self.write_point_group_position_m1m3, ""),
+                ("?POS M2", self.write_point_group_position_m2, ""),
+                ("?STAT", self.write_status, ""),
+            )
+        }
+
+        self.collection_point_regexp = re.compile(r"(?P<group>.*)_P(?P<index>.*)")
+
+        duplicate_keys = self.dispatchers.keys() & self.canned_replies.keys()
+        if duplicate_keys:
+            raise RuntimeError(
+                f"Bug: keys {duplicate_keys} appear in both "
+                "canned_replies and command_handlers"
+            )
+
+        self._laser_warmup_start_tai: None | float = None
+        self._reference_group = "M1M3"
+        # TODO: Get valid working frame values.
+        self.valid_working_frame = {""}
+        # TODO: Get good default working frame value.
+        self.working_frame = ""
+        self.laser_status = "LOFF"
+        self.laser_warmup_time = 60.0
+
+        self.t2sa_status = "READY"
+
+        # How long to wait while pretending to measure (seconds)
+        self.measurement_duration = 2.0
+
+        # Define the "bodies" in the system. Assume m1m3 is at the origin of
+        # the coordinate frame pointing up, m2 is 3 meters away, pointing down
+        # and camera is 2 meters away pointing down.
```

Comment:
I suggest moving this configuration to near the top of this file as one or more named constants. The simplest thing would be to define a constant such as OPTIMAL_POSITION that is set to exactly the dict you have below (and use these comments above it).